src/app/regionen/(index)/_components/RegionListNonpublic.tsx

```diff
@@ -1,3 +1,5 @@
+'use client'
```

Comment:
Those files where our issue the last time, right?
Did you try updating them like in https://blitzjs.com/docs/usage-next-13#blitz-rpc (like updating the import)? Last time we had issues with differences between dev-mode and bundled preview AFAIR.

It would be good to know if those are still there, so we know to not use those features. But also to report them to the Blitz repo.

Reply:
blitz build && blitz start --port 5173 works so seems to be fine